test/drbgtest.c

```diff
@@ -358,16 +358,18 @@ static int error_check(DRBG_SELFTEST_DATA *td)
         goto err;
 
     /* Test insufficient entropy */
+    if (!init(drbg, td, &t))
+        goto err;
     t.entropylen = drbg->min_entropylen - 1;
```

Comment:
t is reset to the defaults in init(), so t.entropylen needs to be set after the init(), otherwise the following generation will succeed.